cachi2/core/package_managers/pip.py

```diff
@@ -460,14 +460,6 @@ def get_version(self) -> Optional[str]:
             log.warning("No project.version in pyproject.toml")
             return None
 
-    def check_dynamic_version(self) -> bool:
```

Comment:
You don't explain anywhere why you're dropping this check.

Reply:
Can I use similar reasoning?

> People should be aware of setup.py soft deprecation by now, do we want to hold everyone's hand? I mean displaying a warning for users who genuinely need setup.py (because pyproject.toml simply doesn't cut it for them as they may have C deps) doesn't feel right. I wouldn't strictly argue against having a warning if you proposed it somewhere in the code, I'm just questioning the usefulness given the circumstances.

Even though it is not deprecated. But the version is optional so I don't see much value in the warning message.